pkg/apis/azuredisk/v1beta2/types.go

```diff
@@ -521,3 +521,84 @@ type ListSnapshotsResult struct {
 	// +optional
 	NextToken string `json:"next_token,omitempty"`
 }
+
+type AzDiskDriverConfiguration struct {
+	Controller ControllerConfiguration `json:"controller"`
+	Node NodeConfiguration `json:"node"`
+	// The namespace where driver related custom resources are created
+	ObjectNamespace string `json:"objectNamespace"`
+	// CSI endpoint
+	Endpoint string `json:"endpoint"`
+	// node id
+	NodeID string `json:"nodeID"`
```

Comment:
`NodeID` is only used by the node driver and can be moved to the `NodeConfiguration`